apps/api/src/server/applications/region/__tests__/get-regions.test.js

```diff
@@ -31,3 +33,28 @@ describe('Get regions', () => {
 		]);
 	});
 });
+
+test('checks cache working for regions', async () => {
+	nodeCache.flushAll();
+
+	const cachedRegion = {
+		id: 1,
+		name: 'cached test',
+		displayNameEn: 'cached test name en',
+		displayNameCy: 'cached test name cy'
+	};
+
+	setCache('regions', [cachedRegion]);
+
+	const resp = await request.get('/applications/region');
+
+	expect(resp.status).toEqual(200);
+	expect(resp.body).toEqual([
+		{
+			id: cachedRegion.id,
+			name: cachedRegion.name,
+			displayNameEn: cachedRegion.displayNameEn,
+			displayNameCy: cachedRegion.displayNameCy
+		}
+	]);
```

Comment:
We can do the same thing here and ensure that `database.region.findMany` has not been called

Reply:
I am not sure what the 'same thing' is?

Reply:
Sorry, I meant that we can test here same as we do in the "get regions" endpoint that we do not make any calls to a database